src/main/java/ch/njol/skript/expressions/ExprInverse.java

```diff
@@ -0,0 +1,63 @@
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+
+@Name("Inverse Boolean")
+@Description("An expression to obtain the inverse value of a boolean")
+@Examples("set {_gravity} to inverse of player's flight mode")
+@Since("INSERT VERSION")
+public class ExprInverse extends SimpleExpression<Boolean> {
+
+	static {
+		Skript.registerExpression(ExprInverse.class, Boolean.class, ExpressionType.COMBINED, 
+			"[the] (inverse|opposite)[s] of %booleans%"
+		);
+	}
+
+	private Expression<Boolean> booleanExpr;
```

Comment:
Could call this just `booleans`

Reply:
I don't think it is a good idea, because at the end, they are still expressions. To avoid confusion, i would recommend keeping this naming.

Do you still want me to rename it?

Reply:
We typically don't include a suffix on the expression fields, so I do still think renaming it would be best. I don't think it introduces any confusion in this case